etl-api/migrations/20250827000000_base.sql

```diff
@@ -0,0 +1,64 @@
+-- Base schema for etl-api
+
+-- Create application schema
+create schema if not exists app;
+
+-- Tenants
+create table app.tenants (
+    id text primary key,
+    name text not null
+);
+
+-- Images
+create table app.images (
+    id bigint generated always as identity primary key,
```

Comment:
Can be shortened everywhere.
```suggestion
    id bigserial primary key,
```

Reply:
There are differences between the two: https://www.naiyerasif.com/post/2024/09/04/stop-using-serial-in-postgres/